syntax = "proto3";

package raft;

message AppendEntriesReq {
}

message AppendEntriesResp {
}

service Raft {
  rpc AppendEntries (AppendEntriesReq) returns (AppendEntriesResp) {}
}
